aiohomekit/controller/ble/pairing.py

```diff
@@ -85,7 +85,10 @@
 
 logger = logging.getLogger(__name__)
 
-DISCOVER_TIMEOUT = 30
+# The discover timeout is how longer we will wait for and advertisement to be
+# received. If we don't get it in this time we will try again later since
+# the scanner is always running anyways.
```

Comment:
Not much point in waiting for something that may never come if the device is sleeping